tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -1259,6 +1236,130 @@ public void handlePath(String xpath) {
             }
         });
 
+        // 'P' Process, like 'p' but without inheritance replacement
+        fixList.add('P', "input-Processor-no-inheritance-replacement", new CLDRFilter() {
+            private DisplayAndInputProcessor inputProcessor;
+
+            @Override
+            public void handleStart() {
+                inputProcessor = new DisplayAndInputProcessor(cldrFileToFilter, true);
+            }
+
+            @Override
+            public void handleEnd() {
+                inputProcessor = null; // clean up, just in case
+            }
+
+            @Override
+            public void handlePath(String xpath) {
+                String value = cldrFileToFilter.getStringValue(xpath);
+                String newValue = inputProcessor.processInput(xpath, value, null);
+                if (value.equals(newValue)) {
+                    return;
+                }
+                String fullXPath = cldrFileToFilter.getFullXPath(xpath);
+                replace(fullXPath, fullXPath, newValue);
+            }
+        });
+
+        // use DAIP for one thing only: replaceBaileyWithInheritanceMarker
+        fixList.add('I', "Inheritance-substitution", new CLDRFilter() {
+            private DisplayAndInputProcessor inputProcessor;
+            private final int STEPS_FROM_ROOT = 1; // only process if locale's level matches; root = 0, en = 1, ...
+
+            @Override
+            public void handleStart() {
+                int steps = stepsFromRoot(cldrFileToFilter.getLocaleID());
```

Comment:
If we fix the inheritance, we may not have to proceed in levels. But we can decide that later, so this is ok.